develop/smart-contracts/wasm-ink.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Wasm (ink!)
+description: Learn to build Wasm smart contracts with ink!, a Rust-based eDSL. Explore installation, contract structure and key features.
+---
+
+# Wasm (ink!)
+
+## Introduction
+
+ink! is an embedded domain-specific language (eDSL) designed to develop [Wasm](https://webassembly.org/){target=\_blank} smart contracts using the Rust programming language. 
+
+Rather than creating a new language, ink! is just standard Rust in a well-defined "contract format" with specialized `#[ink(…)]` attribute macros. These attribute macros tell ink! what the different parts of your Rust smart contract represent and ultimately allow ink! to do all the magic needed to create Polkadot SDK-compatible Wasm bytecode. Because of this, it inherits critical advantages such as:
+
+- Strong memory safety guarantees
+- Advanced type system
+- Comprehensive development tooling
+- Support from Rust's extensive developer community
+
+And because ink! smart contracts are compiled to Wasm, they deliver:
+
+- High execution speed
+- Platform independence
+- Enhanced security through sandboxed execution
+
+These contracts can be deployed on any blockchain built with the Polkadot SDK that implements the [Contracts Pallet](https://docs.rs/pallet-contracts/latest/pallet_contracts/){target=\_blank}.
+
+## Installation
+
+ink! smart contract development requires the installation of [cargo-contract](https://github.com/use-ink/cargo-contract){target=\_blank}, a command-line interface (CLI) tool that provides essential utilities for creating, testing, and managing ink! projects.
+
+For step-by-step installation instructions, including platform-specific requirements and troubleshooting tips, refer to the official cargo-contract [Installation](https://github.com/use-ink/cargo-contract?tab=readme-ov-file#installation){target=\_blank} guide.
+
+## Quick Start
```

Comment:
This is very nitpicking, but I think that we are sticking to "Get Started" for this kind of titles